src/components/Layout/NavBar.tsx

```diff
@@ -8,38 +8,30 @@ import { Br, Strong } from "@/components/Core";
  * Initial draft for the navigation bar; alignment with DSFR guidelines is WIP.
  */
 export const NavBar = () => (
-  <header className="bg-dinum-white-0 overflow-hidden navbar-shadow">
-    <div className="max-w-[78rem] px-[24px] mx-auto flex items-center justify-between p-2  lg:px-8">
-      <Link
-        className="flex items-center gap-10 hover:bg-dinum-white-1 ml-[-0.5rem]  p-2 md:p-4 rounded transition ease-in-out delay-50 duration-300"
-        href="/"
-        title="Retour à l'accueil"
-      >
-        <p className="logo text-[0.7875rem] font-bold leading-[1.0317460317em] tracking-[-.01em] uppercase align-middle inline-block">
-          république
-          <Br className="inline" />
-          française
-        </p>
-        {/* On small screen this section collapse is placed under the navbar */}
-        <div className="hidden sm:flex gap-1 flex-col">
-          <p className="text-xl">
-            <Strong>lasuite.numerique.gouv.</Strong>
-            <i>fr</i>
+  <header className="bg-dinum-white-0 navbar-shadow">
+    <div className="fr-container">
+      <div className="-mx-4 lg:w-fit lg:py-2">
+        <div className="sm:flex items-center fr-enlarge-link hover:bg-dinum-white-1 rounded transition ease-in-out delay-50 duration-300">
+          <p className="p-4 logo text-[0.7875rem] font-bold leading-[1.0317460317em] tracking-[-.01em] uppercase align-middle inline-block">
+            république
+            <Br className="inline" />
+            française
           </p>
-          <p className="font-light text-sm text-dinum-black-1">
-            Un État efficace, simple, souverain grâce au numérique
+          <p className="p-4 border-t border-dinum-grey-4 sm:border-0">
+            <Link
+              className="block text-xl"
+              href="/"
+              aria-label="Accueil - lasuite.numerique.gouv.fr"
+            >
+              <Strong>lasuite.numerique.gouv.</Strong>
+              <i>fr</i>
+            </Link>
```

Comment:
On desktop: Image + Text should be clickable
On mobile: The whole navbar should be clickable.

(feedbacks based on gouvernement's website)


https://github.com/numerique-gouv/lasuite-landingpage/assets/45729124/d0879d65-a6a9-4869-bceb-2088a1321e8f

Reply:
(Precision: It's something that was not functional in my original code)

Reply:
my bad yes, please ignore my comment